mqtt/mqtt-broker/src/session/offline.rs

```diff
@@ -23,6 +23,10 @@ impl OfflineSession {
         self.state.client_id()
     }
 
+    pub fn client_info(&self) -> &ClientInfo {
```

Comment:
q: is it only for me, or it is a bit confusing to have `client_info` for Offline session? worth to rename to `last_client_info` or something like that?

Reply:
don't have strong preferences. We can rename to `last_client_info`